src/main/obsRecorder.ts

```diff
@@ -16,14 +16,19 @@ let obsInitialized = false;
 let timerVideoSourceSize: any;
 // Previous size of the video source as checked by checkVideoSourceSize()
 let lastVideoSourceSize: Size;
+let scene: IScene;
+let recorderOptions: RecorderOptionsType;
+let currentObsTrack: number;
 
 /*
 * Reconfigure the recorder without destroying it.
 */
 const reconfigure = (options: RecorderOptionsType) => {
-  configureOBS(options);
-  const scene = setupScene(options);
-  setupSources(scene, options.audioInputDeviceId, options.audioOutputDeviceId);
+  recorderOptions = options;
+
+  configureOBS();
+  scene = setupScene();
+  setupSources();
```

Comment:
Why do we need this? I think this is the same call we make when we start recording, doesn't this make a reconfigure prevent sleeping?

Reply:
Good catch, yes I believe you're right.